website/docs/configuration/changelog_examples.md

```diff
@@ -0,0 +1,643 @@
+                {% endif %}
+        {% else -%}
+            - {% if commit.breaking %}[**breaking**] {% endif %}{{ commit.message }}
+        {% endif -%}
+    {% endfor -%}
+{% endfor %}
+"""
+
+commit_parsers = [
+  { message = "^feat", group = "added" },
+  { message = "^changed", group = "changed" },
+  { message = "^deprecated", group = "deprecated" },
+  { message = "^fix", group = "fixed" },
+  { message = "^security", group = "security" },
+  { message = "^.*", group = "other" },
+]
+```
+
+</details>
+
+<details>
+  <summary>Raw Output</summary>
+
+```text
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+- Support multiple file formats
+
+### Changed
+
+- Use cache while fetching pages
+
+## [1.0.1](https://github.com/orhun/git-cliff-readme-example/compare/v1.0.0...v1.0.1) - 2021-07-18
+
+### Added
+
+- Add release script
+
+### Changed
+
+- Expose string functions
+
+## [1.0.0] - 2021-07-18
+
+### Added
+
+- Add README.md
+- Add ability to parse arrays
+- Add tested usage example
+
+### Fixed
+
+- Rename help argument due to conflict
+```
+
+</details>
+
+<details>
+  <summary>Rendered Output</summary>
+
+```mdx-code-block
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+- Support multiple file formats
+
+### Changed
+
+- Use cache while fetching pages
+
+## [1.0.1](https://github.com/orhun/git-cliff-readme-example/compare/v1.0.0...v1.0.1) - 2021-07-18
+
+### Added
+
+- Add release script
+
+### Changed
+
+- Expose string functions
+
+## [1.0.0] - 2021-07-18
+
+### Added
+
+- Add README.md
+- Add ability to parse arrays
+- Add tested usage example
+
+### Fixed
+
+- Rename help argument due to conflict
+```
+
+</details>
+
+## Keep a changelog
+
+<details>
+  <summary>TOML configuration</summary>
+
+```toml
+[changelog]
+header = """# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+"""
+
+body = """
+{% if version -%}
+    ## [{{ version | trim_start_matches(pat="v") }}] - {{ timestamp | date(format="%Y-%m-%d") }}
+{% endif -%}
+{% for group, commits in commits | group_by(attribute="group") %}
+    ### {{ group | upper_first }}
+    {% for commit in commits %}
+        - {{ commit.message | upper_first }}\
+    {% endfor %}
+{% endfor %}\n
+"""
+
+footer = """
```

Comment:
release-plz doesn't support the footer. That's because git-cliff only recently added the ability to have templates there so at the beginning I didn't add support for it.
I created https://github.com/MarcoIeni/release-plz/issues/1621

Also, I'm not a fan of commit_parsers. With those commit parsers a commit like `fix: add abc` will be identified as "Changed".

Since (except for the footer part), this Keep A Changelog config is very similar to the default one, imo we can delete this example entirely. What do you think?

Reply:
footer: sorry I didn't know and didn't check. my bad. I don't even like having the footer.

keep a changelog can go then :)

commit_parsers: I am new to this and I liked the bling bling, it's your call tho :)

let me know if I should change it or if you arew taking care of it.

Reply:
done, let me know if you prefer to keep mdx-blocks . they were mostly an annoyance.

Reply:
ps: feel free to change the commit_parsers too, but I think it's good to give alternatives examples otherwise why would even have this section?

Reply:
alternatively: redefining the parsers grouping to your liking?

Reply:
For the mdx-blocks we can disable the lint 👍
I'll think about it

Reply:
let me know, I have currently removed them.